### PR TITLE
Shares from different lots sold on same day not considered replacements

### PR DESCRIPTION
**Pull request by fberger**

### CODE REVIEW

File: tests/same_day_sold_not_replacement.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Cnt,Sym,Desc,BuyDate,Basis,SellDate,Proceeds,AdjCode,Adj,FormPosition,BuyLot,IsReplacement
```
Comment (adlr) [inline tests/same_day_sold_not_replacement.csv:1]: can you also add a text file, similar to the other tests?

File: wash.py [hunk 2]
```diff
@@ -53,12 +53,22 @@ def merge_buy_lots(merge_from, merge_to):
...
+      return False
+    if buy_lots_match(lot, loss):
+      return False
+    if lot.is_replacement:
+      return False
+    if not lot.selldate or lot.selldate > loss.selldate:
+      return True
+    if lot.selldate < loss.selldate:
+      return False
+    # lot.selldate == loss.selldate
```
Comment (adlr) [inline wash.py]: remove this line?

**Comment by adlr** [comment]
Thanks for this patch. If you can fix these I'll pull it in.

In true Google style, can you squash your changes into this commit?

Thanks,
-adlr

File: tests/same_day_sold_not_replacement.csv (hunk 1 above)
Comment (fberger) [inline tests/same_day_sold_not_replacement.csv:1]: Done.

**Comment by fberger** [comment]
Done.

**Comment by adlr** [comment]
Thanks, it's in!

**Pull request merged**